### PR TITLE
Translated API to Ukrainian

### PR DESCRIPTION
**Pull request by YehorBoiar**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

-  Yehor Boiar

## Language: 

- Ukrainian

## Terms defined:

- API

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -1057,7 +1057,13 @@
       Seti ya vipengele na taratibu zinazotolewa na maktaba moja ya programu au 
       huduma ya tovuti ambayo programu nyingine inaweza kuwasiliana nayo. 
       API sio msimbo, hifadhidata, au seva: ni mahali pa ufikiaji.
-      
+  en:
```
Comment (olexandr-konovalov) [inline glossary.yml:1060]: should be "uk"

Comment (olexandr-konovalov) [reply glossary.yml:1060]: As README at https://github.com/carpentries/glosario says, this "must be a single ISO 639 language code" and for Ukrainian language that is "uk"

**Comment by olexandr-konovalov** [comment]
@Dod900ls1 why to close this? If you use Git locally, it's easy to update the PR. You need to change to your branch, edit the file, commit changes. The PR will be updated automatically with a new "fixup" commit. Even better, instead of this, you can also amend the existing commit, and the force-push it _to your feature branch in your fork_

**Comment by Dod900ls1** [comment]
I closed it accidentaly. I tried to rename a branch and it have got deleted in the process, so PR closed automatically. Sorry!

**Review by Herobread — APPROVED** [review]

File: glossary.yml [hunk 2]
```diff
@@ -1057,7 +1057,13 @@
       Seti ya vipengele na taratibu zinazotolewa na maktaba moja ya programu au 
       huduma ya tovuti ambayo programu nyingine inaweza kuwasiliana nayo. 
       API sio msimbo, hifadhidata, au seva: ni mahali pa ufikiaji.
-      
+  uk:
+    term: "Інтерфейс Програмування Додатків (API)"
```
Comment (olexandr-konovalov) [inline glossary.yml:1061]: додатків -> застосунків

Варто пошукати інші переклади та порівняти:
- прикладний програмний інтерфейс
- інтерфейс прикладного програмування

тощо

File: glossary.yml [hunk 3]
```diff
@@ -1057,7 +1057,13 @@
...
+    def: >
+      Набір функцій і процедур, наданих однією програмною бібліотекою або веб-службою,
+      через які інший додаток може з нею взаємодіяти. API – це не код, база даних чи
```
[the remark this answers is not in the document]
Comment (Dod900ls1) [reply glossary.yml:1064]: Good call!

**Comment by YehorBoiar** [comment]
@Herobread @VeronikaShevc Please review

**Review by VeronikaShevc — APPROVED** [review]

File: glossary.yml [hunk 4]
```diff
@@ -1034,6 +1034,12 @@
       Seti ya vipengele na taratibu zinazotolewa na maktaba moja ya programu au
       huduma ya tovuti ambayo programu nyingine inaweza kuwasiliana nayo.
       API sio msimbo, hifadhidata, au seva: ni mahali pa ufikiaji.
+  uk:
+    term: "Інтерфейс Програмування Застосунків (API)"
```
[the remark this answers is not in the document]
Comment (YehorBoiar) [reply glossary.yml:1038]: No, because acronym

[the remark this answers is not in the document]
Comment (Herobread) [reply glossary.yml:1038]: oright meh

[the remark this answers is not in the document]
Comment (olexandr-konovalov) [reply glossary.yml:1038]: Not really: see https://abbrs.info/ - I think @Herobread is right

[the remark this answers is not in the document]
Comment (olexandr-konovalov) [reply glossary.yml:1038]: @YehorBoiar so one more change, and we will merge this one! It's currently the oldest still open PR among Ukrainian translations

**Review by olexandr-konovalov — CHANGES_REQUESTED** [review]
@YehorBoiar one more fix (lowercase) and then this will be ready (submitting this via "review" tool so it will be properly displayed with the "changes requested" status.

**Comment by YehorBoiar** [comment]
@olexandr-konovalov done!

**Review by olexandr-konovalov — APPROVED** [review]
Thanks @YehorBoiar! @elletjies please merge

**Pull request merged**